ui/src/pages/Dashboard/DataIngestionModal.tsx

```diff
@@ -0,0 +1,72 @@
+import Modal from '@splunk/react-ui/Modal';
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+
+import { variables } from '@splunk/themes';
+import Button from '@splunk/react-ui/Button';
+import { StyledButton } from '../EntryPageStyle';
+
+const ModalWrapper = styled(Modal)`
+    width: 60%;
+    height: 80%;
+    margin-top: 4vh;
+`;
+
+const ModalHeader = styled(Modal.Header)`
+    background-color: ${variables.backgroundColorHover};
```

Comment:
I am not sure the semantic name of the color `backgroundColorHover` applies to the non-hover background color